validations/semantic_deprecated_opts_test.go

```diff
@@ -44,6 +45,18 @@ modules:
 		checkDeprecatedOptError(errors[1], gruntOptsYamlField, 17)
 		checkDeprecatedOptError(errors[2], mavenOptsYamlField, 25)
 	})
+
+	It("checkExtDeprecatedOpts - Sanity", func() {
+
+		mta, _ := mta.UnmarshalExt(mtaContent)
```

Comment:
I think this returns an error (due to fields unsupported in mta extension) which you ignore here (also in the test above - it's better to check there was no error because if there was an error you might get partial results)